### PR TITLE
repos: 4.12 RHEL 9: Fix URLs for NFV & FDP repos

### PR DESCRIPTION
**Pull request by travier**
Should unblock https://github.com/openshift/os/pull/905

### CODE REVIEW

**Comment by travier** [comment]
CC @HuijingHei

**Comment by travier** [comment]
CC @mike-nguyen

File: core-services/release-controller/_repos/ocp-4.12-rhel9.repo [hunk 1]
```diff
@@ -36,7 +36,7 @@ failovermethod = priority
 
 [rhel-9-nfv]
 name = rhel-9-nfv
-baseurl = https://cdn.redhat.com/content/eus/rhel9/9.0/x86_64/nfv/os/
+baseurl = https://cdn.redhat.com/content/e4s/rhel9/9.0/x86_64/nfv/os/
```
Comment (cgwalters) [inline core-services/release-controller/_repos/ocp-4.12-rhel9.repo:39]: (wondering what the 4 means)

**Review by cgwalters — APPROVED** [review]

**Pull request merged**